react/fields/Checkbox/Checkbox.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) "~seek-style-guide/theme";
+
+.root {
+  display: inline-block;
+}
+
+.input {
+  opacity: 0;
+  position: absolute;
+}
+
+.label {
+  .touchableText(@standard-type-scale);
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+}
+
+.checkBox {
+  width: @grid-row-height * 3;
+  height: @grid-row-height * 3;
+  border: @field-border-width solid @sk-mid-gray-light;
+  border-radius: @field-border-radius;
+  margin-right: @field-gutter-width;
+  padding: @field-border-radius;
+  fill: @sk-pink;
+
+  .input:not([readonly]):not([disabled]):focus + .label &, .rootFocus & {
+    outline: none;
+    border-color: transparent;
+    .focus();
+  }
+}
+
+.checkMark {
+  display: block;
+  width: initial;
+  height: initial;
```

Comment:
This is to override the icon width and height [set by the Icon component](https://github.com/SEEK-Jobs/seek-style-guide/blob/master/react/icons/icon.less#L9-L10). I would like the icon to fill up the parent checkbox, setting it to ```initial``` will do that without magic numbers.